extensions/adobe/experience/marketo/marketo-contact-lead-person.schema.json

```diff
@@ -0,0 +1,190 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/marketo/marketo-contact-lead-person",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Marketo Contact/Lead Person Schema",
+  "type": "object",
+  "description": "Marketo Person schema for use within  B2B space. Includes other Profile mixins like person/work details, opt in/out etc.",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "definitions": {
+    "marketo-contact-lead-person": {
+      "properties": {
+        "marketo:marketoIntegrationLogID": {
+          "title": "Marketo Integration Log Id",
+          "description": "Marketo integration log id.",
+          "type": "string"
+        },
+        "marketo:additionalFields": {
+          "title": "Additional Fields",
+          "description": "Additional fields.",
+          "type": "string"

```

Comment:
how is this really a string?  What goes here?

Reply:
+1, what is the expected value in this field?